src/core/view/StrokeView.cpp

```diff
@@ -172,10 +185,10 @@ void StrokeView::paint(bool dontRenderEditingStroke, bool markAudioStroke, bool
         cairo_set_operator(crEffective, CAIRO_OPERATOR_SOURCE);
     }
 
-    if (s->getErasable() && !dontRenderEditingStroke) {
+    if (s->getErasable() and !dontRenderEditingStroke) {
         // don't render erasable for previews
         drawErasableStroke(crEffective, s);
-    } else if (s->hasPressure() && !highlighter) {
+    } else if ((s->hasPressure() or s->getLineStyle().hasHeavyDownstroke()) and !highlighter) {
```

Comment:
I think we should keep the && and || symbols here (rather than `or` and `and`). It's not explicitly in the coding conventions, but those keywords don't appear anywhere in the code. You can ignore this comment if you want.